devel/unicorn/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        unicorn-engine unicorn 2.0.0-rc7
+github.tarball_from archive
+
+name                unicorn
+version             2.0.0-rc7
+license             GPL-2+
+categories          devel
+platforms           darwin
```

Comment:
this is the default value for the `platforms` field since MacPorts v2.7 and can, therefore, be removed from the Portfile